scripts/generate_dag.py

```diff
@@ -221,7 +226,60 @@ def print_airflow_variables(dataset_id: str, dag_contents: str, env: str):
         elif var.startswith("value."):
             var = var.replace("value.", "", 1)
         print(f"  - {var}")
-    print()
+        missing_var = var.split(".")[1]
+        user_input = input(f"Airflow Variable Missing | {missing_var} value: ")
+        valid_input = validate_user_input(missing_var, user_input)
+        var_json[dataset_id][missing_var] = valid_input
+
+    print(f"\n{dataset_id} Variables file:")
+    print(f"\n{json.dumps(var_json, indent = 4)}")
+    confirm_input = input(
+        f"\nTo confirm type 'lgtm'. To edit a value, type the variable name: "
+    )
+    var_json = validate_confirmation_input(var_json, dataset_id, confirm_input)
+    return json.dumps(var_json)
+
+
+def validate_user_input(missing_var: str, user_input: str):
+    docker_img_regex = f"(?:gcr.io\/)[a-z\-]*(\/)[a-z\_]*"  # need to update this regex
+    while True:
```

Comment:
Let's try using https://click.palletsprojects.com/ to make the user input handling more elegant.